<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreGUI::ConfirmCloseDialod</class>
 <widget class="QDialog" name="CoreGUI::ConfirmCloseDialod">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm quit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt;
padding: 5px;
</string>
     </property>
     <property name="text">
      <string>Do one of the following on quit:</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveSession">
     <property name="text">
      <string>Do not save files, but save state to be restored on next launch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveFiles">
     <property name="text">
      <string>Save all opened files and close them</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSaveOneFile">
     <property name="text">
      <string>Save unsaved file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDontSave">
     <property name="text">
      <string>Save nothing and quit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel quit</string>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
